srcpkgs/kubernetes-kind/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'kubernetes-kind'
+pkgname=kubernetes-kind
+version=0.6.0
+revision=1
+wrksrc="kind-${version}"
+build_style=go
+go_import_path="sigs.k8s.io/kind"
+hostmakedepends="go"
```

Comment:
Actually not necessary, `build_style=go` adds `go` to `hostmakedepends` automatically.